### DISCUSSION TITLE
EADDRINUSE for port 1338?

### DISCUSSION THREAD

**Discussion by joeyfigaro** [coder/code-server#6769] [Q&A]
I'm not sure where port 1338 is being configured, but it doesn't seem to be something I can change. All mention of configurable ports only touch on the port used in the bind address, which happens to be 8080 by default. 

The workspace I'm looking to open in codeserver includes a backend that uses port 1338, which means I can't run codeserver when the backend is running.

Any ideas?

**Comment by code-asher** [comment]
code-server reads the `PORT` environment variable so maybe that is being set to 1338?  Flags take precedence over environment variables, so you could set `--bind-addr` (or the deprecated `--port`) to override it.

If it is not the environment variable, the other place to check is the config file.  `~/.config/code-server/config.yaml` is the path I believe.

**Reply by joeyfigaro** [reply]
Perfect! Thank you.
